### DISCUSSION TITLE
react-native-bluetooth-serial - this is not working with new architecture enabled on RN version - 0.74.5

### DISCUSSION THREAD

**Discussion by abhayagrawal-fareye** [reactwg/react-native-new-architecture#228] [Libraries]
(NOBRIDGE) ERROR  TypeError: Cannot set property 'on' of null 
<img width="736" alt="Screenshot 2024-09-18 at 7 47 55 PM" src="https://github.com/user-attachments/assets/dda94e4e-d4e2-4a96-a1c3-eeb387a5c7c4">

**Comment by cortinico** [comment]
Have you opened an issue on the `react-native-bluetooth-serial` issue tracker?

**Reply by abhayagrawal-fareye** [reply]
https://github.com/rusel1989/react-native-bluetooth-serial 
No one is maintaining it so I am asking for the solution here
https://www.npmjs.com/package/react-native-bluetooth-serial - It was last modified 7 years ago

**Reply by cortinico** [reply]
Then if it's unmaintained it's unlikely it will ever get New Architecture support.
